src/jit/codegenlegacy.cpp

```diff
@@ -1925,8 +1926,9 @@ void                CodeGen::genRangeCheck(GenTreePtr  oper)
         GenTreeArrLen* arrLenExact = arrLen->AsArrLen();
         lenOffset = arrLenExact->ArrLenOffset();
 
-        // We always load the length into a register on ARM and x64.
 #if !CPU_LOAD_STORE_ARCH && !defined(_TARGET_64BIT_)
+        // We always load the length into a register on ARM and x64.
```

Comment:
This one is pretty weird because it's talking about ARM and x64, but it is now in a clause that excludes those. Not sure how much it matters in codegenlegacy, though ...